### ISSUE TITLE
Achievement level 5 bug

### ISSUE THREAD

**Issue by SeravySensei** [rathena/rathena#4141]
* **Description of Issue**: 
  * Result: 
  * Expected Result: 
  * How to Reproduce: 
  * Official Information:
  

Level 5 achievement is never triggered. Caused by the requirements being in wrong order.
https://github.com/rathena/rathena/blob/7f772c32d3be201861946bb64720c231828465ac/src/map/achievement.cpp#L700

Level 5 is higher than level 6 so it gets skipped.  The numbers 135 and 104 should be swapped, assuming they are correct in the first place, they might not be. 135 and 140 are so close it's suspicious.

Furthermore the data for level 9 and higher is incorrect, the numbers seem to be placeholders. TODO Comment mentions level 8 or higher which further implies the 140 in the list being an additional, mistakenly added element.

Finally, the algorithm should be modified to award the first incomplete achievement found, not the last. Otherwise if enough points are obtained (or the server admin changes rewards) at the same time, in-between levels are never awarded.

Note : I won't submit a pull request for this one myself because I don't know the correct amounts of required achievement points.

**Comment by aleos89** [comment]
This would be directly related to #4131. Now I understand what that report is trying to convey.

This issue isn't out fault. When grabbing packet data when I was working on this system, Gravity literally passes those values. For some reason their level 6 value dropped and then it increased again. I just implemented it as I found it. The rest of the levels are TODO because we didn't have any packet data beyond those levels to know their real values since the system was new and not many people had gotten that high. Maybe Gravity fixed the ranks by now. I'll have to try and grab new data.
